templates/taxi/car_list.html

```diff
@@ -1,16 +1,37 @@
 {% extends "base.html" %}
 
 {% block content %}
-  <h1>Car list</h1>
+  <h1>
+    Car list
+    <a href="{% url 'taxi:car-create' %}">+</a>
+  </h1>
   {% if car_list %}
-    <ul>
-      {% for car in car_list %}
-        <li>
-          <a href="{% url "taxi:car-detail" pk=car.id %} ">{{ car.id }}</a>
-          {{ car.model }} ({{ car.manufacturer.name }})
-        </li>
-      {% endfor %}
-    </ul>
+    <table class="table">
+        <tr>
+          <th>ID</th>
+          <th>Model</th>
+          <th>Update</th>
+          <th>Delete</th>
+        </tr>
+
+        {% for car in car_list %}
+          <tr>
+            <td>
+                {{ car.id }}
+            </td>
+            <td>
+                {{ car.model }}
+            </td>
+            <td>
+              <a href="{% url "taxi:car-update" pk=car.id %}"></a>
+            </td>
+            <td>
+              <a href="{% url "taxi:car-delete" pk=car.id %}"></a>
```

Comment:
Add something into `a` tag, because there is no clickable object on the website.